src/test/CropManager-unit.t.sol

```diff
@@ -395,6 +449,35 @@ contract CropManagerTest is TestBase {
         a.flux(address(b), address(a), 100 * 1e18);
     }
 
+    function test_flux_from_other() public {
+        (Usr a, Usr b) = init_user();
+        a.join(100 * 1e6);
+        assertEq(a.gems(), 100 * 1e18);
+        assertEq(a.stake(), 100 * 1e18);
+        a.allow(address(b));
+        b.flux(address(a), address(b), 100 * 1e18);
+        assertEq(b.gems(), 100 * 1e18);
+        assertEq(b.stake(), 100 * 1e18);
```

Comment:
I would assert corresponding decreases in `gems` and `stake` for `a` for completeness.

Reply:
Done!